tools/PI/DevHome.PI/BarWindowHorizontal.xaml

```diff
@@ -0,0 +1,126 @@
+    xmlns:winex="using:WinUIEx"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevHome.PI.Controls"
+    xmlns:helpers="using:DevHome.PI.Helpers"
+    mc:Ignorable="d"
+    Title="" MinHeight="70" MinWidth="700" Height="70" Width="700" MaxHeight="70"
+    TaskBarIcon="Images/pi.ico"
+    Closed="WindowEx_Closed">
+
+    <!-- TODO Make DesktopAcrylicBackdrop/MicaBackdrop and AcrylicBackgroundFillColorBaseBrush user-configurable.-->
+    <Window.SystemBackdrop>
+        <MicaBackdrop/>
+    </Window.SystemBackdrop>
+
+    <Grid x:Name="MainPanel" Padding="0" Loaded="MainPanel_Loaded" Background="Transparent" SizeChanged="MainPanel_SizeChanged">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <!--Chrome buttons for a horizontal bar -->
+        <Grid x:Name="HorizontalBar" Background="Transparent" VerticalAlignment="Center" Height="28" Grid.Row="0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition x:Name="LeftPaddingColumn" Width="0"/>
+                <ColumnDefinition x:Name="ExtraSystemButtons" Width="*"/>
+                <ColumnDefinition x:Name="RightPaddingColumn" Width="0"/>
+            </Grid.ColumnDefinitions>
+
+            <TextBlock x:Name="TitleTextBlock" Text="DevHome PI" FontSize="10" Margin="10,0,0,0" HorizontalAlignment="Left" Grid.Column="1" VerticalAlignment="Center"/>
+            <StackPanel x:Name="ChromeButtonPanel" Orientation="Horizontal" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="0, 0, 10, 0"  Background="Transparent" Grid.Column="1">
+                <Button
+                      x:Uid="SnapButton"
+                      x:Name="SnapButton" Style="{StaticResource ChromeButton}" FontSize="11"
+                      Click="{x:Bind viewModel.PerformSnapCommand}" HorizontalAlignment="Left" IsEnabled="{x:Bind viewModel.IsSnappingEnabled, Mode=OneWay}">
+                    <TextBlock Text="{x:Bind viewModel.CurrentSnapButtonText, Mode=OneWay}"/>
+                </Button>
+            </StackPanel>
+        </Grid>
+
+        <StackPanel x:Name="AllControls" Grid.Row="1" Orientation="Horizontal" HorizontalAlignment="Left" Background="Transparent" >
+            <StackPanel.Resources>
+                <Style TargetType="TextBlock">
+                    <Setter Property="FontFamily" Value="Segoe Fluent Icons"/>
+                    <Setter Property="FontSize" Value="20"/>
+                </Style>
+                <Style TargetType="Button">
+                    <Setter Property="BorderThickness" Value="0"/>
+                    <Setter Property="Background" Value="Transparent"/>
+                </Style>
+                <Style TargetType="controls:ProcessSelectionButton">
```

Comment:
Nice thing about it being here is that all the styles for the buttons are in one place... if you changed the style for button, hopefully you'll change the style for ProcessSelectionButton at the same time since it's right next to it.